roles/robot-pkgs/tasks/main.yml

```diff
@@ -3,12 +3,12 @@
 - name: Add ROS key
   apt_key:
     keyserver: ha.pool.sks-keyservers.net
-    id: 421C365BD9FF1F717815A3895523BAEEB01FA116
+    id: C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
     state: present
-- name: Add ROS repository
+- name: Add ROS repositories
   template:
     src: ros-packages.j2
-    dest: /etc/apt/sources.list.d/ros-packages.list
+    dest: /etc/apt/sources.list.d/ros-package-repositories.list
```

Comment:
This change will actually result in an issue for existing users. Rather than replacing the file, it will create a new one. This will result in two identical files with different names. Even though the current name isn't ideal, the extra effort to fix it isn't really worth it.

But thank you for pointing it out! Perhaps if we phase out 354, we can make sure to fix this if it ever gets phased back in.

Reply:
OK, thanks for pointing that out. I will be sure to not include this change in my forked version, If there are not objections, I will go ahead and open new pull request from my forked repo, to fix this issue for current users.